maquetado/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Título del libro</title>
+    <link rel="stylesheet"  type="text/css" href="styles.css">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+   
+</head>
+
+<body>
+<div class="book">
+<div class="book-cover">
+<img  height= "368px" width= "261px" src="book-cover.png" alt="Portada de libro">
+</div>
```

Comment:
Use indentation like this:

```suggestion
<body>
  <div class="book">
    <div class="book-cover">
      <img  height= "368px" width= "261px" src="book-cover.png" alt="Portada de libro">
    </div>
```

That way the HTML will be more readable 😄